docs/changelogs/ChangeLog-1.4.X.md

```diff
@@ -1862,7 +1862,7 @@
 
 - [`KT-27362`](https://youtrack.jetbrains.com/issue/KT-27362) Anonymous classes representing function/property references contain rarely used methods
 - [`KT-35626`](https://youtrack.jetbrains.com/issue/KT-35626) NI: Performance problem with many type parameters
-- [`KT-36047`](https://youtrack.jetbrains.com/issue/KT-36047) Compiler produces if-chain instead of switch when when subject captured as variable
+- [`KT-36047`](https://youtrack.jetbrains.com/issue/KT-36047) Compiler produces if-chain instead of switch when subject captured as variable
```

Comment:
This is about the case where the subject of `when` is captured as a variable, so the original wording ("... when when subject ...") was actually correct.